README.md

```diff
@@ -1 +1,94 @@
-# Youtube Thumbnail Downloader
+<h1 align="center">Youtube Thumbnail Downloader</h1>
+
+  <p align="center">
+    Youtube Thumbnail Downloader is a Simple Web page Tool, which is used to download the thumbnail of any youtube video.   
```

Comment:
It would be great, If you add the Live URL of project

Reply:
Will try to do that